Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -1201,4 +1201,11 @@ theorem inner_matrix_col_col [Fintype m] (A B : Matrix m n 𝕜) (i j : n) :
     ⟪Aᵀ i, Bᵀ j⟫ₑ = (Aᴴ * B) i j := by
   simp [PiLp.inner_apply, dotProduct, mul_apply', mul_comm]
 
+/-- The matrix representation of `innerSL 𝕜 x` given by an orthonormal basis `b` and `b₂`
+is equal to `vecMulVec (star b₂) (star (b.repr x))`. -/
+theorem toMatrix_innerSL_apply [DecidableEq ι] {ι₂ : Type*} [Fintype ι₂]
+    (b : OrthonormalBasis ι 𝕜 E) (b₂ : OrthonormalBasis ι₂ 𝕜 𝕜) (x : E) :
+    (innerSL 𝕜 x).toMatrix b.toBasis b₂.toBasis = vecMulVec (star b₂) (star (b.repr x)) := by
```

Comment:
Can you use the existing `m` and `n` variables here instead of `ι`? Or do they have the wrong assumptions?